### ISSUE TITLE
Intoeing

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7397]
**Preferred term label:**
Intoeing

**Synonyms**
pigeon-toeing; in-toeing.

**Definition (free text, please give PubMed ID)**
Internal rotation of the long axis of the foot to the line of progression that causes foot to point inward (PMIDs: 24075135; 29763169)

**Parent term (use hpo.jax.org/app)**
Abnormality of toe HP:0001780

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by jagcsun** [comment]
There are three common [causes](https://www.rch.org.au/uploadedFiles/Main/Content/rheumatology/intoeing.pdf) of intoeing, for which HPO terms are already available: 
1. Metatarsus adductus HP:0001840
2. Tibial torsion HP:0100694
3. Abnormal femoral torsion HP:0031069

**Comment by pnrobinson** [comment]
@jagcsun 
I am not sure we gain much by adding another term, as the three terms have more detail. I think we need to discuss items like this when hopefully we next have a skeletal workshop.
